packages/stylelint-plugin/package.json

```diff
@@ -19,7 +19,7 @@
     "lodash.flatmap": "^4.5.0",
     "lodash.kebabcase": "^4.1.1",
     "nano-memoize": "^2.0.0",
-    "postcss": "^8.4.20",
+    "postcss": "8.4.19",
```

Comment:
Rather than locking the version, perhaps you could just downgrade and not lock it.

Renovate would create a new upgrade PR and it would fail CI once your new stylelint config has updated.
However, there would also be the chance that the other deps get updated and the versions become realigned, which then it wouldn't be a problem to upgrade.

Reply:
Yeah, not locking it works but I figured it was safer to lock, can unlock.